libs/eo/landing-page/shell/src/lib/eo-landing-page-footer.component.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * @license
+ * Copyright 2020 Energinet DataHub A/S
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License2");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  ChangeDetectionStrategy,
+  Component,
+  NgModule,
+  ViewEncapsulation,
+} from '@angular/core';
+
+const selector = 'eo-landing-page-footer';
+
+@Component({
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None,
+  selector,
+  styles: [
+    `
+      @use '@energinet-datahub/watt/utils' as watt;
+      ${selector} {
+        display: grid;
+        grid-template-columns: repeat(3, 1fr);
+        width: 100%;
+        box-shadow: 0px 0px 15px rgba(0, 0, 0, 0.1);
+        background: var(--watt-color-neutral-white);
+        padding: var(--watt-space-l) var(--watt-space-m); // No mixing exists for "space-inset-stretch-l", see: C:lib/foundations/spacing/_spacing.import.scss
+
+        .${selector}__a {
+          text-decoration: underline; // This overrides WATT styles (which removes the underline on a tags)
+          color: var(
+            --watt-color-primary
+          ); // This overrides the '--watt-color-primary-dark' color which is added by a class on the element
```

Comment:
question: Why is this different between what we need as per the Figma design sketch and what Watt prescribes? Is it because of misalignment between Watt's Figma and Storybook?